src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hosting/ConfigurableLanguageServerFeatureOptions.cs

```diff
@@ -39,6 +40,7 @@ internal class ConfigurableLanguageServerFeatureOptions : LanguageServerFeatureO
     public override bool DisableRazorLanguageServer => _disableRazorLanguageServer ?? _defaults.DisableRazorLanguageServer;
     public override bool ForceRuntimeCodeGeneration => _forceRuntimeCodeGeneration ?? _defaults.ForceRuntimeCodeGeneration;
     public override bool UseNewFormattingEngine => _useNewFormattingEngine ?? _defaults.UseNewFormattingEngine;
+    public override bool AvoidExplicitCommitCharacters => _avoidExplicitCommitCharacters ?? _defaults.AvoidExplicitCommitCharacters;
```

Comment:
I have a small concern around the naming here. On my initial read through, I thought we might be disabling all explicit commit characters based on the name, but the fix is actually more scoped than that. Consider making this name more specific or (probably better) add a good doc comment.

Reply:
I was figuring we may find more cases when we need to avoid explicit commit characters, but since this PR only addresses one specific case, I followed your suggestion and made option name more specific, as well as made doc-comment in the base class more specific as well.

Reply:
I think Andrew has a very good suggestion for the name that addresses your concern as well.